LoopKitUI/HUDProvider.swift

```diff
@@ -9,7 +9,8 @@
 import Foundation
 
 public enum HUDTapAction {
-    case presentViewController(UIViewController & CompletionNotifying)
+    case presentPumpViewController(UIViewController & CompletionNotifying)
+    case presentCGMViewController(UIViewController & CompletionNotifying & GlucoseUnitObserver)
```

Comment:
I think it would be better to not require CGMViewController to conform to GlucoseUnitObserver, but rather make it optional, and the receiver of this action can test via typecasting to see if the VC conforms.

Reply:
My concern with this is that it isn't at all clear which objects support `GlucoseUnitObserver`. Any optional check for a protocol conformance is hidden from the API contract. A comment would help, but comments go stale (and we'd need to remember to put it whenever there is optional protocol conformance).

Reply:
updated as suggested